src/Analyzers/Analyzers/test/StartupAnalyzerTest.cs

```diff
@@ -4,347 +4,527 @@
 using System.Collections.Concurrent;
 using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Analyzer.Testing;
 using Microsoft.CodeAnalysis;
 using Xunit;
+using VerifyCS = Microsoft.AspNetCore.Analyzers.CSharpCodeFixVerifier<
+    Microsoft.AspNetCore.Analyzers.StartupAnalyzer,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
 
 namespace Microsoft.AspNetCore.Analyzers
 {
-    public class StartupAnalyzerTest : AnalyzerTestBase
+    public class StartupAnalyzerTest
     {
-        public StartupAnalyzerTest()
+        [Fact]
+        public async Task StartupAnalyzer_FindsStartupMethods_StartupSignatures_Standard()
+        {
+            var source = @"// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
📝 The sources have been inlined. In the past, analyzer testing was prone to false negatives when syntax or semantic errors were present in the test sources, and separation of the sources into files part of the normal compilation helped avoid this. With Microsoft.CodeAnalysis.Testing, this is no longer a concern; the test will fail if any non-declared compilation error appears in the source.

Reply:
How are these things related? This seems like a change for the sake of change if we like things the way they are. 

I prefer the code inline, but I'm not sure what other people who work on this part of the code like.

Reply:
There are some real disadvantages to putting the code inline, like losing syntax highlighting, and having to escape double-quotes.

Reply:
There are also some advantages to having them inline, specifically the ability to use the full markup syntax and the ability to write tests for analyzer behavior in the presence of syntax errors in the input and/or output.